Convert `osu!standard` > `osu!` everywhere

DoryanR opened the pull request:
This big change was initiated by @peppy via [#5245 (comment)](https://github.com/ppy/osu-wiki/pull/5245#discussion_r611390728).

I took the initiative to make this change, which took a lot of time and energy.

List of things done on this PR:

- Renamed `osu!standard` to `osu!` (including images), and in some cases called `osu! mode`.
      This was only done on the English version (see for outdated other files?)
- Added `outdated: true` where necessary.
- As some articles have not been updated for a long time, and as a check of the files has been set up, I did the necessary to make it pass the checks.
- In some cases, I modified the links to other articles, which did not work.
- I made various fixes on other articles (especially typo).

**Normally**, all the old terms have been changed, except for the names of some tournaments, I've gone through all the wiki pages 3-4 times, but I don't exclude forgetting.

However, this term persists in the news (which I have absolutely not touched), in the disambiguation (osu!) of other languages, in the redirections (not knowing if I should remove them or not, I leave them), and in the name of a tournament.

For the things not concerning the change of the game mode name, I did what I saw, because I'll be honest I didn't have the energy to check EVERYTHING in detail.

TODO:

- [x] Fix all checks errors 
- [x] Doing a total revision to better adapt the term `osu!` to the different situations, following the different suggestions proposed during the reviews
- [x] Change the abbreviations for the mode icons (o!s, o!m, o!t, o!c) to their full names (like [here](https://github.com/ppy/osu-wiki/pull/5245/commits/b77f0e63eb323d84c6c101bf4bad192b0a07274b)) on each article.

TicClick commented on wiki/Accuracy/en.md:
```diff
@@ -8,11 +8,11 @@ Accuracy is a percentile measurement of a player's ability to hit [hit objects](
 ## Game modes
 
-### ![](/wiki/shared/mode/osu.png) osu!standard
+### ![](/wiki/shared/mode/osu.png) osu! mode
```
changes like this are excessive -- it's clear that `osu!` as mentioned here is a game mode, not the game in general. most of the time you want it to be `osu!standard -> osu!` kind of fix

TicClick commented on wiki/Accuracy/en.md:
```diff
@@ -8,11 +8,11 @@ Accuracy is a percentile measurement of a player's ability to hit [hit objects](
-### ![](/wiki/shared/mode/osu.png) osu!standard
+### ![](/wiki/shared/mode/osu.png) osu! mode
 
-![Accuracy = (50 \* number of 50s + 100 \* number of 100s + 300 \* number of 300s) / 300(number of 0s + number of 50s + number of 100s + number of 300s)](img/accuracy_standard.png "Accuracy formula for osu!standard")
```
regarding the file name, are there any other files with exclamation mark in name on osu! wiki at all? if not, I'd be cautious and leave this as `accuracy_osu.png`

TicClick commented on wiki/Article_styling_criteria/Writing/ru.md:
```diff
@@ -152,7 +152,7 @@
 Названия игровых режимов нужно писать следующим образом:
 
-- `osu!standard` (неофициальное название для устранения неоднозначности);
+- `osu!` (неофициальное название для устранения неоднозначности);
```
I'll take care of this translation in a separate PR

TicClick commented on wiki/Contests/en.md:
```diff
@@ -10,21 +10,21 @@ A list of contests that are not mentioned on the [Community Contests page](https
-| 5 | [Rameses B - Flaklypa](https://osu.ppy.sh/beatmapsets/190390) | ![osu!standard][o!s] | [-kevincela-](https://osu.ppy.sh/users/266596) | [Results](https://osu.ppy.sh/home/news/2014-07-14-monthly-beatmapping-contest-5-results) | [Score #5](https://osu.ppy.sh/p/contestresults?c=11) |
-| 6 (Aspire) | [LeaF - Evanescent](https://osu.ppy.sh/beatmapsets/227126) | ![osu!standard][o!s] | [Charles445](https://osu.ppy.sh/users/85000) | [Results](https://osu.ppy.sh/home/news/2014-11-13-monthly-beatmapping-contest-aspire-results) | [Score #6](https://osu.ppy.sh/p/contestresults?c=12) |
-| 7.1 | [Soleily - Renatus](https://osu.ppy.sh/beatmapsets/241526) | ![osu!standard][o!s] | [Gamu](https://osu.ppy.sh/users/611174) | [Results](https://osu.ppy.sh/home/news/2015-03-04-monthly-beatmapping-contest-7-results) | [Score #7.1](https://osu.ppy.sh/p/contestresults?c=13) |
+| 1 | [Rostik - Liquid (Paul Rosenthal Remix)](https://osu.ppy.sh/beatmapsets/123593) | ![osu! mode][o!s] | [Charles445](https://osu.ppy.sh/users/85000) | [Results](https://osu.ppy.sh/home/news/2013-10-20-monthly-beatmapping-contest-1-results) | [Score #1](https://osu.ppy.sh/p/contestresults?c=7) |
```
on listings like these, better keep it as `osu!` as well -- it's clear it's a mode

TicClick commented on wiki/Disambiguation/osu!/en.md:
```diff
@@ -3,5 +3,5 @@
 
 - The game and its community
-- The default [game mode](/wiki/Game_mode), sometimes referred to as *[osu!standard](/wiki/Game_mode/osu!)*
+- The default [game mode](/wiki/Game_mode), previously known as *[osu!standard](/wiki/Game_mode/osu!)*
```
has it really ever been known as such? think it's best if you remove everything after comma:

```suggestion
- The [default game mode](/wiki/Game_mode/osu!)
```

peppy commented on wiki/Article_styling_criteria/Writing/en.md:
```diff
@@ -177,7 +177,7 @@ October 25, 2016 (11:45 UTC)
 Game modes must be written as follows:
 
-- `osu!standard` (unofficial, but used to prevent ambiguity)
+- `osu!` (unofficial, but used to prevent ambiguity)
```
wrong

DoryanR replied:
The file about ASC has been merged with #5246, the syntax should be good.

peppy commented on wiki/Article_styling_criteria/Writing/en.md:
````diff
@@ -83,7 +83,7 @@ When writing out game modifiers for tournament articles, they must instead use u
 
 ```markdown
-In osu!standard, beatmaps are composed of three different gameplay elements: hit circles, sliders, and spinners.
+In the osu! mode, beatmaps are composed of three different gameplay elements: hit circles, sliders, and spinners.
````
unnecessary

DoryanR replied:
The file about ASC has been merged with #5246, the syntax should be good.

peppy commented on wiki/Beatmap_Editor/Compose/en.md:
```diff
@@ -91,7 +92,7 @@ Hold `Alt` to trigger Distance Snap. The value can range from x0.1 - x2.0.
 **Your construction toolkit, made for DIY (do it yourself).**
 
-**Please note that in osu!standard's default notes hitsounds works differently in osu!mania.**
+**Please note that in the osu!'s mode default notes hitsounds works differently in osu!mania.**
```
This isn't good as an english sentence. The original *and* this change are wrong.

peppy commented on wiki/Beatmap_Editor/Song_Setup/en.md:
```diff
@@ -237,17 +237,17 @@ You can, however, include the skin elements inside the beatmap folder. As most c
 Moving the slider towards the right will space out the "closeness" of hit objects in-time before stacking will occur. This option is best left as is to its default value of `7`.
 
-*Notice: Stacked hit circles in osu!standard will move towards the point where the stacking occurs.*
+*Notice: Stacked hit circles in the osu! mode will move towards the point where the stacking occurs.*
```
just `osu!` is fine here. In fact I would probably just have regex'd this whole change and not used the "mode" suffix anywhere. Reviewing this PR is going to be as large a pain as it was for you to seemingly manually make these changes.

DoryanR replied:
Yes indeed, I'm afraid I'm going to have to change everything to make it perfect, I thought so.

But the worry is that if you only put `osu!` for the main game mode, because of the way the sentences are written, which is not explicit to differentiate the game mode and the name of the game, you might get confused.

Unless you take some of the sentences to adapt the game mode (which I think is the best thing to do), it will be complicated to adapt it otherwise.

Then the idea of putting instead `osu! game mode` in some cases only, as @Nivalyx did [here](https://github.com/ppy/osu-wiki/pull/5246/commits/248c743f29b59db0131f988263e1dbd2646655b0) would it be possible ?

peppy replied:
"hit circles" are only present in osu! mode. also, in this context i'm pretty sure the mention of the mode could be removed altogether.

in fact this whole sentence is just unneccesary and reads bad, so it wasn't a great example.

peppy replied:
The linked example could probably just read "In osu! mode" or just osu!, dunno.

DoryanR replied:
I will see to not put everywhere `osu! mode`, because the problem is that it is everywhere, and after thinking it is not applicable everywhere, as in the example you gave.

So after that, I will see to adapt it better, which should make it better I think.

peppy commented on wiki/Beatmap_Editor/Song_Setup/en.md:
```diff
@@ -237,17 +237,17 @@ You can, however, include the skin elements inside the beatmap folder. As most c
 ### Mode Specific
 
-`Allowed Modes` is a dropdown to confirm that the beatmap can only be played by the specified mode. Default setting is `All`, which means the creator will create an osu!standard beatmap that can be converted into the other game modes. This can be changed to be osu!taiko-specific, osu!catch-specific, or osu!mania-specific; but keep in mind that the change will remove every hit object in the beatma ...
+`Allowed Modes` is a dropdown to confirm that the beatmap can only be played by the specified mode. Default setting is `All`, which means the creator will create an osu! mode beatmap that can be converted into the other game modes. This can be changed to be osu!taiko-specific, osu!catch-specific, or osu!mania-specific; but keep in mind that the change will remove every hit object in the beatmap.
```
again unnecessary mode suffix. p robably the case with the majority.

peppy commented on wiki/Beatmapping/Overall_difficulty/en.md:
```diff
@@ -27,7 +27,7 @@ If the player hits outside of the 50's hit window, it will count as a miss.
 ## Sliders and spinners
 
-In [osu!standard](/wiki/Game_mode/osu!), sliders will reward a 300 as long as they are hit within the 50's hit window. This is sometimes referred to as *slideracc* and is removed in ScoreV2<!-- TODO: link -->.
+In the [osu! mode](/wiki/Game_mode/osu!), sliders will reward a 300 as long as they are hit within the 50's hit window. This is sometimes referred to as *slideracc* and is removed in ScoreV2<!-- TODO: link -->.
```
Also unnecessary. Just osu! is fine..

peppy reviewed (dismissed):
Not going to go through this whole diff, but general direction seems wrong.

DoryanR commented:
- I changed `osu! mode` to `osu!`, and in some cases to `osu! game mode` or to `osu! game modes` when several mod names are present in the same sentence.
- I have applied TicClick's modifications and these tips, as well as peppy's tips and modifications.
- I changed the abbreviations of the modes (o!s, o!t, o!c and o!m) by their full names, for more clarity for the translators.
- Changed some links that were dead.
- I changed the images name in the shared folder that had old terms, and fixed all the redirects.

Walavouchey commented on wiki/Beatmap_Editor/Compose/en.md:
```diff
@@ -91,7 +92,7 @@ Hold `Alt` to trigger Distance Snap. The value can range from x0.1 - x2.0.
 **Your construction toolkit, made for DIY (do it yourself).**
 
-**Please note that in osu!standard's default notes hitsounds works differently in osu!mania.**
+**Please note that in the osu!'s default notes, hitsounds works differently in osu!mania.**
```
This still doesn't quite work

```suggestion
**Please note that in osu!, default note hitsounds work differently than in osu!mania.**
```

DoryanR replied:
I had not seen him, thank you

peppy commented:
I'm not sure anyone is going to review this. When making huge batch changes like this, each change should be split into its own commit, hopefully with the `sed` command (or otherwise) used to generate the change.

Else you will enter a deadlock caused by no one being willing to read through 6k LoC change.

TicClick commented:
it's ok I'm halfway through

TicClick commented on wiki/Game_modifier/Score_multiplier/en.md:
```diff
@@ -18,9 +18,9 @@ tags:
+For example, if a player plays a [beatmap](/wiki/Beatmap) on osu! game mode with no mods enabled, and gains a final score of 100,000, then the final score will be 100,000. However, if that same player achieved the *exact same play* on the *exact same beatmap*, with the [Hard Rock](/wiki/Game_modifier/Hard_Rock) mod enabled, then the final score would be 106,000 because the Hard Rock mod's Score M ...
 
-However, if multiple mods are enabled, the score multiplier value would be the two score multiplier values of each mod, multiplies together. For example, if the Hard Rock mod and [Half Time](/wiki/Game_modifier/Half_Time) mod are enabled simultaneously in osu!standard, the final value would be approximately 0.32x.
+However, if multiple mods are enabled, the score multiplier value would be the two score multiplier values of each mod, multiplies together. For example, if the Hard Rock mod and [Half Time](/wiki/Game_modifier/Half_Time) mod are enabled simultaneously in osu! game mode, the final value would be approximately 0.32x.
```
it doesn't really matter here
```suggestion
However, if multiple mods are enabled, the score multiplier value would be the two score multiplier values of each mod, multiplies together. For example, if the Hard Rock mod and [Half Time](/wiki/Game_modifier/Half_Time) mod are enabled simultaneously in osu!, the final value would be approximately 0.32x.
```

TicClick commented on wiki/Game_modifier/Spun_Out/en.md:
```diff
@@ -21,12 +21,12 @@ tags:
 ## Description
 
-The **Spun Out** mod is a [game modifier](/wiki/Game_modifier) that is aims to aid [osu!standard](/wiki/Game_mode/osu!) players in successfully spin spinners in a [beatmap](/wiki/Beatmap). The mod will clear all of the beatmap's spinner(s) at exactly 286.48 spins per minute (SPM) in a clockwise rotation, which is 3/5 of Auto's 477.26 SPM.
+The **Spun Out** mod is a [game modifier](/wiki/Game_modifier) that is aims to aid [osu!](/wiki/Game_mode/osu!) game mode players in successfully spin spinners in a [beatmap](/wiki/Beatmap). The mod will clear all of the beatmap's spinner(s) at exactly 286.48 spins per minute (SPM) in a clockwise rotation, which is 3/5 of Auto's 477.26 SPM.
```
```suggestion
The **Spun Out** mod is a [game modifier](/wiki/Game_modifier) that is aims to aid [osu!](/wiki/Game_mode/osu!) players in successfully spin spinners in a [beatmap](/wiki/Beatmap). The mod will clear all of the beatmap's spinner(s) at exactly 286.48 spins per minute (SPM) in a clockwise rotation, which is 3/5 of Auto's 477.26 SPM.
```

TicClick commented on wiki/Game_modifier/Sudden_Death/en.md:
```diff
@@ -45,10 +45,10 @@ This mod has the same effect across all [game modes](/wiki/Game_mode).
 
-- Missing a note with the Sudden Death mod enabled will result in an extra large miss figure to appear in [osu!standard](/wiki/Game_mode/osu!)
-- When playing with Sudden Death mod enabled in osu!standard mode, there are rare cases where the player may survive from a full life bar drain for breaking the combo count. If the player managed to preserve at least a sliver of health by getting a slider tick in a very short time-margin right after the combo broke and before the life bar is drained completely, the player can continue playing the ...
+- Missing a note with the Sudden Death mod enabled will result in an extra large miss figure to appear in the [osu!](/wiki/Game_mode/osu!) game mode.
+- When playing with Sudden Death mod enabled in osu! game mode, there are rare cases where the player may survive from a full life bar drain for breaking the combo count. If the player managed to preserve at least a sliver of health by getting a slider tick in a very short time-margin right after the combo broke and before the life bar is drained completely, the player can continue playing the be ...
```
```suggestion
- Missing a note with the Sudden Death mod enabled will result in an extra large miss figure to appear in [osu!](/wiki/Game_mode/osu!).
- When playing with Sudden Death mod enabled in osu!, there are rare cases where the player may survive from a full life bar drain for breaking the combo count. If the player managed to preserve at least a sliver of health by getting a slider tick in a very short time-margin right after the combo broke and before the life bar is drained completely, the player can continue playing the beatmap as is and no large miss figure will be imposed.
```

TicClick commented on wiki/Game_modifier/Summary/en.md:
```diff
@@ -4,39 +4,39 @@
 ## xK mod score multipliers
 
-*Notice: This table only affects beatmaps converted from osu!standard to osu!mania during play, not on osu!mania-specific beatmaps!*
+*Notice: This table only affects beatmaps converted from osu! game mode to osu!mania during play, not on osu!mania-specific beatmaps!*
```
```suggestion
*Notice: This table only affects beatmaps converted from osu! to osu!mania during play, not on osu!mania-specific beatmaps!*
```

TicClick commented on wiki/Game_modifier/Summary/en.md:
```diff
@@ -47,7 +47,7 @@
 ### *x*K mod with Co-op score multipliers
 
-*Notice: This table only affects beatmaps converted from osu!standard to osu!mania during play, not on osu!mania-specific beatmaps. And as such, the Co-Op mod gives no penalty on osu!mania-specific beatmaps.*
+*Notice: This table only affects beatmaps converted from osu! game mode to osu!mania during play, not on osu!mania-specific beatmaps. And as such, the Co-Op mod gives no penalty on osu!mania-specific beatmaps.*
```
```suggestion
*Notice: This table only affects beatmaps converted from osu! to osu!mania during play, not on osu!mania-specific beatmaps. And as such, the Co-Op mod gives no penalty on osu!mania-specific beatmaps.*
```

TicClick commented on wiki/Game_modifier/Target_Practice/en.md:
```diff
@@ -32,11 +32,11 @@ During a play with the Target Practice mod enabled, players will be presented wi
 The mod's use is mostly for fun, but can also be used to practice maintaining a consistent tempo.
 
-![TP Gameplay](img/TP-gameplay.jpg "Gameplay of osu!standard with the Target Practice mod enabled")
+![TP Gameplay](img/TP-gameplay.jpg "Gameplay of osu! game mode with the Target Practice mod enabled")
```
```suggestion
![TP Gameplay](img/TP-gameplay.jpg "osu! gameplay with the Target Practice mod enabled")
```

TicClick commented on wiki/Gameplay/Input_device/Graphics_tablet/en.md:
```diff
@@ -8,7 +8,7 @@ stub: true
 ![](img/tablet.jpg "Graphics tablet")
 
-A **graphics tablet**, or just *tablet*, is an input device that is originally intended for digital drawing and artistry, but can be used as a way to control cursor movement in [osu!standard](/wiki/Game_mode/osu!). It is one of the most common play styles in osu!standard.
+A **graphics tablet**, or just *tablet*, is an input device that is originally intended for digital drawing and artistry, but can be used as a way to control cursor movement in [osu! mode](/wiki/Game_mode/osu!) game mode. It is one of the most common play styles in osu!.
```
```suggestion
A **graphics tablet**, or just *tablet*, is an input device that is originally intended for digital drawing and artistry, but can be used as a way to control cursor movement in [osu!](/wiki/Game_mode/osu!). It is one of the most common play styles in osu!.
```

TicClick commented on wiki/Gameplay/Input_device/Wiimote/en.md:
```diff
@@ -6,7 +6,7 @@ stub: true
 ![](img/wiimote.jpg "Wiimote")
 
-**Wiimote** is the primary game controller for Nintendo [Wii](https://en.wikipedia.org/wiki/Wii "Wikipedia"). Although  mostly seen as gimmicky or silly, the Wiimote can be used in osu!standard to aim using the sensor bar. You can rebind any button to click in the osu! options.
+**Wiimote** is the primary game controller for Nintendo [Wii](https://en.wikipedia.org/wiki/Wii "Wikipedia"). Although  mostly seen as gimmicky or silly, the Wiimote can be used in osu! game mode to aim using the sensor bar. You can rebind any button to click in the osu! options.
```
```suggestion
**Wiimote** is the primary game controller for Nintendo [Wii](https://en.wikipedia.org/wiki/Wii "Wikipedia"). Although  mostly seen as gimmicky or silly, the Wiimote can be used in osu! to aim using the sensor bar. You can rebind any button to click in the osu! options.
```

TicClick commented on wiki/Glossary/Combo_milestone/en.md:
```diff
@@ -6,6 +6,6 @@ tags:
 # Combo milestone
 
-**Combo milestones** are a specific set of [combos](/wiki/Glossary/Combo_(score_multiplier)) that trigger the game to shoot stars and display a comboburst (if enabled in the [options](/wiki/Options)). The combo milestones for [osu!standard](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch) are 30x, 60x, 100x, and multiples of 50 after 100. For [osu!taiko](/wiki/Game_mode/osu!taiko) ...
+**Combo milestones** are a specific set of [combos](/wiki/Glossary/Combo_(score_multiplier)) that trigger the game to shoot stars and display a comboburst (if enabled in the [options](/wiki/Options)). The combo milestones for [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch) game modes are 30x, 60x, 100x, and multiples of 50 after 100. For [osu!taiko](/wiki/Game_mode/osu!tai ...
```
```suggestion
**Combo milestones** are a specific set of [combos](/wiki/Glossary/Combo_(score_multiplier)) that trigger the game to shoot stars and display a comboburst (if enabled in the [options](/wiki/Options)). The combo milestones for [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch) are 30x, 60x, 100x, and multiples of 50 after 100. For [osu!taiko](/wiki/Game_mode/osu!taiko), the combo milestones are multiples of 50. For [osu!mania](/wiki/Game_mode/osu!mania), the combo milestones are multiples of 100.
```

TicClick commented on wiki/Glossary/Combo_multiplier_effect/en.md:
```diff
@@ -8,6 +8,6 @@ tags:
 The **combo multiplier effect** is a bug with [scoreV1](/wiki/Score/ScoreV1) which allows the [score](/wiki/Score) to count backwards. This is a flaw with the 32-bit signed integer where (in computing) the max integer is `2,147,483,647`. Once that cap is reached, the score will begin to count backwards. [ScoreV2](/wiki/Score#scorev2) fixes this problem by capping the score at 1 million points, no ...
 
-The combo multiplier effect occurs in [osu!standard](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), and [osu!catch](/wiki/Game_mode/osu!catch). This occurs because the listed game modes use the player's current [combo](/wiki/Glossary/Combo_(score_multiplier)) as a part of the score calculations. Meaning that a player would get a higher score if they obtained a [full combo (FC)](/w ...
+The combo multiplier effect occurs in [osu!](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), and [osu!catch](/wiki/Game_mode/osu!catch) game modes. This occurs because the listed game modes use the player's current [combo](/wiki/Glossary/Combo_(score_multiplier)) as a part of the score calculations. Meaning that a player would get a higher score if they obtained a [full combo (FC)] ...
```
```suggestion
The combo multiplier effect occurs in [osu!](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), and [osu!catch](/wiki/Game_mode/osu!catch). This occurs because the listed game modes use the player's current [combo](/wiki/Glossary/Combo_(score_multiplier)) as a part of the score calculations. Meaning that a player would get a higher score if they obtained a [full combo (FC)](/wiki/Glossary#fc) than someone who played the same map with a broken combo.
```

TicClick commented on wiki/Glossary/Comboburst/en.md:
```diff
@@ -9,11 +9,11 @@ tags:
 *Combo bursts* celebrate achieving a [combo milestone](/wiki/Glossary/Combo_milestone) by flashing a combo burst image. Said images are different for each game mode.
 
-- In [osu!standard](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch), a character will burst out from either the left or right side of the window. By default, said character will be [pippi](/wiki/Mascots#-pippi) and [Yuzu](/wiki/Mascots#-yuzu) respectively.
+- In [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch) game modes, a character will burst out from either the left or right side of the window. By default, said character will be [pippi](/wiki/Mascots#-pippi) and [Yuzu](/wiki/Mascots#-yuzu) respectively.
```
```suggestion
- In [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch), a character will burst out from either the left or right side of the window. By default, said character will be [pippi](/wiki/Mascots#-pippi) and [Yuzu](/wiki/Mascots#-yuzu) respectively.
```

TicClick commented on wiki/Glossary/Comboburst/en.md:
```diff
@@ -9,11 +9,11 @@ tags:
 - In [osu!mania](/wiki/Game_mode/osu!mania), a character will burst from the right side of the stage by default.
 
-For osu!standard, osu!taiko, and osu!catch, stars will shoot out from either the left or right side of the playfield for combos less than 100x, then both sides simultaneously for combos 100x or higher. For osu!mania, the stars will shoot out from the left and right sides of the stage.
+For osu!, osu!taiko, and osu!catch game modes, stars will shoot out from either the left or right side of the playfield for combos less than 100x, then both sides simultaneously for combos 100x or higher. For osu!mania, the stars will shoot out from the left and right sides of the stage.
```
```suggestion
For osu!, osu!taiko, and osu!catch, stars will shoot out from either the left or right side of the playfield for combos less than 100x, then both sides simultaneously for combos 100x or higher. For osu!mania, the stars will shoot out from the left and right sides of the stage.
```

TicClick commented on wiki/Glossary/Health_bar/en.md:
```diff
@@ -13,12 +13,12 @@ tags:
 The **health bar** (also known as the *HP bar*) is a bar which displays a player's remaining [health](/wiki/Beatmapping/Health). In all [game modes](/wiki/Game_mode) expect [osu!mania](/wiki/Game_mode/osu!mania), there is an indicator moving along the health bar that shows a player's current health.
 
-In [osu!standard](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch), if the health bar drains completely, the player will fail the current [beatmap](/wiki/Beatmap) (assuming no special game modifiers are enabled).
+In [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch) game modes, if the health bar drains completely, the player will fail the current [beatmap](/wiki/Beatmap) (assuming no special game modifiers are enabled).
```
```suggestion
In [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch), if the health bar drains completely, the player will fail the current [beatmap](/wiki/Beatmap) (assuming no special game modifiers are enabled).
```

TicClick commented on wiki/Glossary/Health_bar/en.md:
```diff
@@ -13,12 +13,12 @@ tags:
 In [osu!taiko](/wiki/Game_mode/osu!taiko), it is impossible to fail a beatmap *during* play, but it is possible to fail after the beatmap has ended if the player has not filled the health bar enough.
 
-In osu!standard, osu!catch, and osu!taiko, the health bar is displayed in the top-left corner of the screen. Whereas in osu!mania, it is rotated 90 degrees counterclockwise, and is placed to the right of the playfield.
+In osu!, osu!catch, and osu!taiko game modes, the health bar is displayed in the top-left corner of the screen. Whereas in osu!mania, it is rotated 90 degrees counterclockwise, and is placed to the right of the playfield.
```
```suggestion
In osu!, osu!catch, and osu!taiko, the health bar is displayed in the top-left corner of the screen. Whereas in osu!mania, it is rotated 90 degrees counterclockwise, and is placed to the right of the playfield.
```

TicClick commented on wiki/Glossary/Health_bar/en.md:
```diff
@@ -13,12 +13,12 @@ tags:
+In osu!, osu!catch, and osu!taiko game modes, the health bar is displayed in the top-left corner of the screen. Whereas in osu!mania, it is rotated 90 degrees counterclockwise, and is placed to the right of the playfield.
 
-In osu!standard and osu!catch, the health bar will drain at a constant rate throughout the course of a beatmap, with the exception of breaks. This effect is known as [health drain](/wiki/Beatmapping/Health_drain).
+In osu! and osu!catch game modes, the health bar will drain at a constant rate throughout the course of a beatmap, with the exception of breaks. This effect is known as [health drain](/wiki/Beatmapping/Health_drain).
```
```suggestion
In osu! and osu!catch, the health bar will drain at a constant rate throughout the course of a beatmap, with the exception of breaks. This effect is known as [health drain](/wiki/Beatmapping/Health_drain).
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -146,7 +146,7 @@ Combo has two meanings:
 2. The number representing the number of objects a player has hit consecutively without missing or [sliderbreaking](#sliderbreak) once.
    - This number is displayed in various places:
-     - Bottom-left in [osu!standard](/wiki/Game_mode/osu!).
+     - Bottom-left in[osu!](/wiki/Game_mode/osu!) game mode.
```
```suggestion
     - Bottom-left in[osu!](/wiki/Game_mode/osu!).
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -177,11 +177,11 @@ This feature was later disabled on 2013-03-05 due to performance concerns.
 A set of numbers that will shoot stars and display a comboburst (if enabled in the [options](/wiki/options)) after obtaining one of these milestones.
 
-The combo milestones for osu!standard and osu!catch are 30, 60, 100, and multiples of 50 after 100. For osu!taiko, the combo milestones are multiples of 50. For osu!mania, the combo milestones are multiples of 100.
+The combo milestones for osu! and osu!catch game modes are 30, 60, 100, and multiples of 50 after 100. For osu!taiko, the combo milestones are multiples of 50. For osu!mania, the combo milestones are multiples of 100.
```
```suggestion
The combo milestones for osu! and osu!catch are 30, 60, 100, and multiples of 50 after 100. For osu!taiko, the combo milestones are multiples of 50. For osu!mania, the combo milestones are multiples of 100.
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -177,11 +177,11 @@ This feature was later disabled on 2013-03-05 due to performance concerns.
 ### Combo Multiplier Effect
 
-The combo multiplier effect occurs in the osu!standard, osu!taiko, and osu!catch game modes. This occurs because said game modes use the player's current combo as a part of the score calculations. Meaning that a player would get a higher score, if they had gotten a full combo, than someone who played the same map with a broken combo.
+The combo multiplier effect occurs in the osu!, osu!taiko, and osu!catch game modes. This occurs because said game modes use the player's current combo as a part of the score calculations. Meaning that a player would get a higher score, if they had gotten a full combo, than someone who played the same map with a broken combo.
```
```suggestion
The combo multiplier effect occurs in the osu!, osu!taiko, and osu!catch. This occurs because said game modes use the player's current combo as a part of the score calculations. Meaning that a player would get a higher score, if they had gotten a full combo, than someone who played the same map with a broken combo.
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -193,11 +193,11 @@ A term used when the player loses their combo during any part of playing a beatm
 A celebration of achieving a [combo milestone](#combo-milestone).
 
-- For osu!standard and osu!catch, a character will *burst* out from either the left or right side of the window.
+- For osu! and osu!catch game modes, a character will *burst* out from either the left or right side of the window.
```
```suggestion
- For osu! and osu!catch, a character will *burst* out from either the left or right side of the window.
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -193,11 +193,11 @@ A term used when the player loses their combo during any part of playing a beatm
 - For osu!mania, a character will *burst* from the right side of the stage (or left side or both if specified in the `skin.ini`).
 
-In addition to this, for osu!standard, osu!taiko, and osu!catch, stars will shoot out from either the left or right side of the playfield, for combos less than 100, then both sides for combos 100 or higher. For osu!mania, the stars will shoot out from the left and right sides of the stage.
+In addition to this, osu!, osu!taiko, and osu!catch game modes, stars will shoot out from either the left or right side of the playfield, for combos less than 100, then both sides for combos 100 or higher. For osu!mania, the stars will shoot out from the left and right sides of the stage.
```
```suggestion
In addition to this, osu!, osu!taiko, and osu!catch, stars will shoot out from either the left or right side of the playfield, for combos less than 100, then both sides for combos 100 or higher. For osu!mania, the stars will shoot out from the left and right sides of the stage.
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -251,7 +251,7 @@ A collective term for *[Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Os
 A set of rhythm games, developed by [iNiS](https://en.wikipedia.org/wiki/INiS) for the Nintendo DS handheld video game system, which share a lot of gameplay and presentation elements.
 
-The osu!standard game mode was is mostly based off of those games.
+The osu! game mode was is mostly based off of those games.
```
```suggestion
The osu! game mode is mostly based off of those games.
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -271,7 +271,7 @@ Fail has three meanings:
 - The incompletion of a beatmap due to:
   - Not filling the health bar up to 80% when completing a beatmap (in [osu!taiko](/wiki/Game_mode/osu!taiko)).
-  - Empty health bar ([osu!standard](/wiki/Game_mode/osu!), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!mania](/wiki/Game_mode/osu!mania)).
+  - Empty health bar ([osu!](/wiki/Game_mode/osu!), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!mania](/wiki/Game_mode/osu!mania) game mode).
```
```suggestion
  - Empty health bar ([osu!](/wiki/Game_mode/osu!), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!mania](/wiki/Game_mode/osu!mania)).
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -285,7 +285,7 @@ Fail has three meanings:
 A term used when a player completes a beatmap while obtaining the maximum combo possible. This does not account accuracy as it is often used to mean clearing a beatmap without having any misses or [sliderbreaks](#sliderbreak).
 
-Due to the [combo multiplier effect](#combo-multiplier-effect), full combos will give the most score in the osu!standard, osu!taiko, and osu!catch modes.
+Due to the [combo multiplier effect](#combo-multiplier-effect), full combos will give the most score in osu!, osu!taiko, and osu!catch game modes.
```
```suggestion
Due to the [combo multiplier effect](#combo-multiplier-effect), full combos will give the most score in osu!, osu!taiko, and osu!catch.
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -370,7 +370,7 @@ A blacklist of words (or users) that the player does not want to be displayed in
 ### Juice stream
 
-An osu!catch element that includes Drops and Droplets. Drops give a score of 100, equivalent to slider ticks in osu!standard, while Droplets give a score of 10, equivalent to the [slider tick](#slider-tick) in osu!standard.
+An osu!catch element that includes Drops and Droplets. Drops give a score of 100, equivalent to slider ticks in osu! game mode, while Droplets give a score of 10, equivalent to the [slider tick](#slider-tick) in osu!.
```
```suggestion
An osu!catch element that includes Drops and Droplets. Drops give a score of 100, equivalent to slider ticks in osu!, while Droplets give a score of 10, equivalent to the [slider tick](#slider-tick) in osu!.
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -426,13 +426,13 @@ A 3 second time period of silence that is sometimes automatically be inserted be
 ### Life Bar
 
-The life bar, also known as the *health bar*, *HP bar*, *spirit bar*, or *ki bar*, is a bar at the top left of the playing screen for [osu!standard](/wiki/Game_mode/osu!), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!taiko](/wiki/Game_mode/osu!taiko). For [osu!mania](/wiki/Game_mode/osu!mania), the life bar is rotated 90 degrees anti-clockwise and is placed next to the bottom-right of the sta ...
+The life bar, also known as the *health bar*, *HP bar*, *spirit bar*, or *ki bar*, is a bar at the top left of the playing screen for [osu!](/wiki/Game_mode/osu!), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!taiko](/wiki/Game_mode/osu!taiko) game modes. For [osu!mania](/wiki/Game_mode/osu!mania), the life bar is rotated 90 degrees anti-clockwise and is placed next to the bottom-right of the  ...
```
```suggestion
The life bar, also known as the *health bar*, *HP bar*, *spirit bar*, or *ki bar*, is a bar at the top left of the playing screen for [osu!](/wiki/Game_mode/osu!), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!taiko](/wiki/Game_mode/osu!taiko). For [osu!mania](/wiki/Game_mode/osu!mania), the life bar is rotated 90 degrees anti-clockwise and is placed next to the bottom-right of the stage.
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -426,13 +426,13 @@ A 3 second time period of silence that is sometimes automatically be inserted be
 In all modes, expect for osu!mania, there is a symbol moving along it indicating the player's life. If the bar is completely empty, the player fails the current beatmap (assuming no special game modifiers are enabled).
 
-osu!standard and osu!catch will have health drain at a constant rate—known as *passive HP*—throughout the course of a beatmap, except during breaks.
+osu! and osu!catch game modes will have health drain at a constant rate—known as *passive HP*—throughout the course of a beatmap, except during breaks.
```
```suggestion
osu! and osu!catch will have health drain at a constant rate—known as *passive HP*—throughout the course of a beatmap, except during breaks.
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -426,13 +426,13 @@ A 3 second time period of silence that is sometimes automatically be inserted be
+osu! and osu!catch game modes will have health drain at a constant rate—known as *passive HP*—throughout the course of a beatmap, except during breaks.
 
-For osu!standard, the bar is replenished by the player [tapping](#tap) hit circles, following sliders and slider ticks, and spinning spinners. The more accurate the tapping, the bigger the boost. The player also receives larger boosts if he achieves a [katu](#katu) or an [geki](#geki) at the end of a combo.
+For osu! game mode, the bar is replenished by the player [tapping](#tap) hit circles, following sliders and slider ticks, and spinning spinners. The more accurate the tapping, the bigger the boost. The player also receives larger boosts if he achieves a [katu](#katu) or an [geki](#geki) at the end of a combo.
```
```suggestion
For osu!, the bar is replenished by the player [tapping](#tap) hit circles, following sliders and slider ticks, and spinning spinners. The more accurate the tapping, the bigger the boost. The player also receives larger boosts if he achieves a [katu](#katu) or an [geki](#geki) at the end of a combo.
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -712,7 +712,7 @@ Silences lasts for 4 weeks before being reset, along with the time that gets add
 ### Skin
 
-A set of graphics and custom sample sets (audio clips) that can replace the default ones used by osu!. This will allow for a more customised user experience. Most of what you see and hear in osu! can be skinned.
+A set of graphics and custom sample sets (audio clips) that can replace the default ones used by osu!. This will allow for a more customized user experience. Most of what you see and hear in osu! can be skinned.
```
this is an alternative spelling
```suggestion
A set of graphics and custom sample sets (audio clips) that can replace the default ones used by osu!. This will allow for a more customised user experience. Most of what you see and hear in osu! can be skinned.
```

TicClick commented on wiki/Glossary/en.md:
```diff
@@ -755,9 +755,9 @@ Stacking hit circles (a clear violation of beat spacing) is, nevertheless, permi
 
 ### Standard
```
the whole section needs to be removed -- it is essentially describing the thing we are removing right now

TicClick commented on wiki/Guides/How_to_Use_the_Offset_Wizard/fr.md:
```diff
@@ -1,23 +0,0 @@
-# Comment utiliser l'assistant de décalage audio
```
why are you removing these files? they need to be marked with `outdated: true` in a separate PR instead

TicClick commented on wiki/Guides/osu!mania_Mapping_Guide/en.md:
```diff
@@ -93,7 +93,7 @@ Alright! Let's drag the `.mp3` of the song we want to map into osu! and a new ma
 | :-- | :-- |
 | Stacking: | This point has no effect for osu!mania so no need to change anything here. |
-| Allowed Modes: | With this point, you change the mode of the editor to the one you want to use for your map. This guide is about osu!mania so we use "osu!mania" of course. Should you choose "All", your editor will be set to osu!standard.  **Changing this option while in an existing osu!mania map will overwrite the map.** |
+| Allowed Modes: | With this point, you change the mode of the editor to the one you want to use for your map. This guide is about osu!mania so we use "osu!mania" of course. Should you choose "All", your editor will be set to osu! game mode.  **Changing this option while in an existing osu!mania map will overwrite the map.** |
```
```suggestion
| Allowed Modes: | With this point, you change the mode of the editor to the one you want to use for your map. This guide is about osu!mania so we use "osu!mania" of course. Should you choose "All", your editor will be set to osu!. **Changing this option while in an existing osu!mania map will overwrite the map.** |
```

TicClick commented on wiki/History_of_osu!/2016/en.md:
```diff
@@ -10,7 +10,7 @@ Links:
 ## February
 
-A sneak peek of the osu!next design was on display in the 46th issue of osu!weekly. peppy blogged various improvements for the game. Most notably, the skin preview in the [options](/wiki/Options) and new skinnable interface sounds were added. The [Best of 2015](/wiki/Contests) results were in with [Panda Eyes & Teminite - Highscore (CiiyusMiapah)](https://osu.ppy.sh/beatmapsets/332532#osu/736215) ...
+A sneak peek of the osu!next design was on display in the 46th issue of osu!weekly. peppy blogged various improvements for the game. Most notably, the skin preview in the [options](/wiki/Options) and new skinnable interface sounds were added. The [Best of 2015](/wiki/Contests) results were in with [Panda Eyes & Teminite - Highscore (CiiyusMiapah)](https://osu.ppy.sh/beatmapsets/332532#osu/736215) ...
```
```suggestion
A sneak peek of the osu!next design was on display in the 46th issue of osu!weekly. peppy blogged various improvements for the game. Most notably, the skin preview in the [options](/wiki/Options) and new skinnable interface sounds were added. The [Best of 2015](/wiki/Contests) results were in with [Panda Eyes & Teminite - Highscore (CiiyusMiapah)](https://osu.ppy.sh/beatmapsets/332532#osu/736215), [cillia - Fairytale, (mintong89)](https://osu.ppy.sh/beatmapsets/275331#taiko/624447), [Halozy - Genryuu Kaiko (Yukina)](https://osu.ppy.sh/beatmapsets/253969#fruits/582801), and [D(ABE3) - MANIERA (Jinjin)](https://osu.ppy.sh/beatmapsets/272871#mania/646681) taking in the most votes for the osu!, osu!taiko, osu!catch, and osu!mania respectively. New items were added to the osu!store; such as: the osu!mug (a coffee mug), the osu!mug with pippi (ditto), and osu!go (a USB thumb drive with osu! pre-installed). Thanks to [LiquidPL](https://osu.ppy.sh/users/5044384), reording of the sections, on the new website, in your profile was now possible.
```

TicClick commented on wiki/History_of_osu!/2018/en.md:
```diff
@@ -40,7 +40,7 @@ In osu!lazer, the infamous catmull styled sliders were implemented; thus allowin
 In osu!web, the total play time was implemented in user profiles. Elasticsearch was updated. The old site was depreciated almost entirely (a new banner is now present almost everywhere on the old site, linking users to the new site). Genres and languages were added to beatmap pages. Recent activity now shows the "show more" option. Searching users was improved. Icons for the forum topics were cha ...
 
-The Community Choice 2017 results were released with Doormat, Backfire, Ascendance, and Protastic101 winning the community's vote in osu!standard, osu!taiko, osu!catch, and osu!mania respectively.
+The Community Choice 2017 results were released with Doormat, Backfire, Ascendance, and Protastic101 winning the community's vote in osu!, osu!taiko, osu!catch, and osu!mania game modes respectively.
```
```suggestion
The Community Choice 2017 results were released with Doormat, Backfire, Ascendance, and Protastic101 winning the community's vote in osu!, osu!taiko, osu!catch, and osu!mania respectively.
```

TicClick commented on wiki/Hit_object/Juice_stream/en.md:
```diff
@@ -6,6 +6,6 @@ stub: true
 *See also: [Score](/wiki/Score)*
 
-**Juice streams** are an element in [osu!catch](/wiki/Game_Modes/osu!catch) which include both drops and droplets. Drops give a score of 100, equivalent to slider ticks in [osu!standard](/wiki/Game_Modes/osu!), while droplets give a score of 10, equivalent to the slider tick in osu!standard.
+**Juice streams** are an element in [osu!catch](/wiki/Game_Modes/osu!catch) which include both drops and droplets. Drops give a score of 100, equivalent to slider ticks in [osu!](/wiki/Game_Modes/osu!), while droplets give a score of 10, equivalent to the slider tick in osu! game mode.
```
```suggestion
**Juice streams** are an element in [osu!catch](/wiki/Game_Modes/osu!catch) which include both drops and droplets. Drops give a score of 100, equivalent to slider ticks in [osu!](/wiki/Game_Modes/osu!), while droplets give a score of 10, equivalent to the slider tick in osu!.
```

TicClick commented on wiki/Mapping_Techniques/Time-distance_equality/en.md:
```diff
@@ -7,6 +7,6 @@ tags:
 # Time-distance equality
 
-**Time-distance equality** is an [osu!standard](/wiki/Game_mode/osu!) [mapping technique](/wiki/Mapping_Techniques) where spacing between [hit objects](/wiki/Hit_object) on the playfield is proportional to the time between them. This is automatically calculated by the [distance snap](/wiki/Beatmap_Editor/Distance_Snap) feature of the [beatmap editor](/wiki/Beatmap_Editor).
+**Time-distance equality** is an [osu!](/wiki/Game_mode/osu!) game mode [mapping technique](/wiki/Mapping_Techniques) where spacing between [hit objects](/wiki/Hit_object) on the playfield is proportional to the time between them. This is automatically calculated by the [distance snap](/wiki/Beatmap_Editor/Distance_Snap) feature of the [beatmap editor](/wiki/Beatmap_Editor).
```
```suggestion
**Time-distance equality** is an [osu!](/wiki/Game_mode/osu!) [mapping technique](/wiki/Mapping_Techniques) where spacing between [hit objects](/wiki/Hit_object) on the playfield is proportional to the time between them. This is automatically calculated by the [distance snap](/wiki/Beatmap_Editor/Distance_Snap) feature of the [beatmap editor](/wiki/Beatmap_Editor).
```

TicClick commented on wiki/Medals/en.md:
```diff
@@ -10,12 +11,12 @@ Medals, previously known as *achievements*, are badges on a user's profile which
 ## Beatmap Packs
 
-The **Beatmap Packs** medals can be obtained by **playing every beatmap all the way through at least once in any difficulty** within one [Theme Beatmap Pack](https://osu.ppy.sh/beatmaps/packs?type=theme). You must play in [osu!standard](/wiki/osu!standard) to qualify a **`CLEARED`** status at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=theme) and the badge.
+The **Beatmap Packs** medals can be obtained by **playing every beatmap all the way through at least once in any difficulty** within one [Theme Beatmap Pack](https://osu.ppy.sh/beatmaps/packs?type=theme). You must play in [osu!](/wiki/Game_mode/osu!) game mode to qualify a **`CLEARED`** status at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=theme) and the badge.
```
```suggestion
The **Beatmap Packs** medals can be obtained by **playing every beatmap all the way through at least once in any difficulty** within one [Theme Beatmap Pack](https://osu.ppy.sh/beatmaps/packs?type=theme). You must play in the [osu!](/wiki/Game_mode/osu!) game mode to qualify a **`CLEARED`** status at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=theme) and the badge.
```

TicClick commented on wiki/Medals/en.md:
```diff
@@ -46,7 +47,7 @@ The star-based medals are split into two groups, *Pass* and *S*. The *Pass* meda
 ## Dedication
 
-The **Dedication** medals can be obtained by playing a lot of osu!standard, collecting many fruits in osu!catch, hitting getting a lot of drum hits in osu!taiko, by pressing a lot of keys in osu!mania.
+The **Dedication** medals can be obtained by playing a lot of osu! game mode, collecting many fruits in osu!catch, hitting getting a lot of drum hits in osu!taiko, by pressing a lot of keys in osu!mania.
```
```suggestion
The **Dedication** medals can be obtained by playing a lot of osu!, collecting many fruits in osu!catch, hitting getting a lot of drum hits in osu!taiko, by pressing a lot of keys in osu!mania.
```

TicClick commented on wiki/Options/en.md:
```diff
@@ -426,6 +426,6 @@ b{YYYY}{MM}{DD}.{revision}{type}
 
 - The old options screen was an actual screen that had tabs, buttons, and a dark pale blue background.
-- The old options screen also featured a skin selection screen that also allows you to preview live play of a beatmap in osu!standard.
+- The old options screen also featured a skin selection screen that also allows you to preview live play of a beatmap in osu! game mode .
```
```suggestion
- The old options screen also featured a skin selection screen that also allows you to preview live play of a beatmap in osu!.
```

TicClick commented on wiki/People/Users_with_unique_titles/en.md:
```diff
@@ -181,7 +181,7 @@ These players earned their titles through either a crazy gameplay achievement or
 | ![][flag_AU] [Jaye](https://osu.ppy.sh/users/4841352) | Drum Decimator | Scored FCs on all Ranked osu!taiko maps as of 3 November 2019 |
 | ![][flag_NL] [Shirotora_old](https://osu.ppy.sh/users/11820) | Bested the Reaper's Game | *Undocumented* <!-- TODO: a reference to TWEWY's 104 building, because he got to rank #104 in a certain amt of time.. need to investigate further --> |
-| ![][flag_US] [xasuma](https://osu.ppy.sh/users/3172980) | The First Completionist | Played all Ranked osu!standard maps as of 27 April 2019 |
+| ![][flag_US] [xasuma](https://osu.ppy.sh/users/3172980) | The First Completionist | Played all Ranked osu! game mode maps as of 27 April 2019 |
```
```suggestion
| ![][flag_US] [xasuma](https://osu.ppy.sh/users/3172980) | The First Completionist | Played all Ranked osu! maps as of 27 April 2019 |
```

TicClick commented on wiki/Performance_points/en.md:
```diff
@@ -52,7 +52,7 @@ Regarding the formula above, *p* represents each score's full pp value (pre-weig
 Elements like [approach rate](/wiki/Beatmapping/Approach_rate) and certain [mods](/wiki/Game_modifier) (namely [Flashlight](/wiki/Game_modifier/Flashlight), [Hidden](/wiki/Game_modifier/Hidden) and [Hard Rock](/wiki/Game_modifier/Hard_Rock)) make navigating the cursor quickly and accurately significantly more difficult, and thus influence the amount of pp a score gives.
 
-In the case of [osu!standard](/wiki/Game_mode/osu!), beatmaps with very large [jumps](/wiki/Beatmap/Pattern/Jump) are considered to be "high aim" beatmaps, and are thus often given very high pp scores. Likewise, beatmaps with more hyperdashing in [osu!catch](/wiki/Game_mode/osu!catch) will be considered similarly. Aim is not considered in gamemodes like [osu!taiko](/wiki/Game_mode/osu!taiko) and  ...
+In the case of [osu!](/wiki/Game_mode/osu!) game mode, beatmaps with very large [jumps](/wiki/Beatmap/Pattern/Jump) are considered to be "high aim" beatmaps, and are thus often given very high pp scores. Likewise, beatmaps with more hyperdashing in [osu!catch](/wiki/Game_mode/osu!catch) will be considered similarly. Aim is not considered in gamemodes like [osu!taiko](/wiki/Game_mode/osu!taiko) an ...
```
```suggestion
In case of [osu!](/wiki/Game_mode/osu!), beatmaps with very large [jumps](/wiki/Beatmap/Pattern/Jump) are considered to be "high aim" beatmaps, and are thus often given very high pp scores. Likewise, beatmaps with more hyperdashing in [osu!catch](/wiki/Game_mode/osu!catch) will be considered similarly. Aim is not considered in gamemodes like [osu!taiko](/wiki/Game_mode/osu!taiko) and [osu!mania](/wiki/Game_mode/osu!mania).
```

TicClick commented on wiki/Play_style/Alternating/en.md:
```diff
@@ -12,7 +12,7 @@ tags:
 *Not to be confused with [Double tapping](/wiki/Play_style/Double_tapping).*
 
-Alternating is a tapping style for [osu!standard](/wiki/Game_mode/osu!) in which players click circles, slide sliders, and spin spinners by alternating between two set keys on their keyboard (`Z` and `X` by default) every other [hit object](/wiki/Hit_object) or so. Hence the name. This style is most common amongst more seasoned players and is one of the hardest to adapt to.
+Alternating is a tapping style for [osu!](/wiki/Game_mode/osu!) game mode in which players click circles, slide sliders, and spin spinners by alternating between two set keys on their keyboard (`Z` and `X` by default) every other [hit object](/wiki/Hit_object) or so. Hence the name. This style is most common amongst more seasoned players and is one of the hardest to adapt to.
```
```suggestion
Alternating is a tapping style for [osu!](/wiki/Game_mode/osu!) in which players click circles, slide sliders, and spin spinners by alternating between two set keys on their keyboard (`Z` and `X` by default) every other [hit object](/wiki/Hit_object) or so. Hence the name. This style is most common amongst more seasoned players and is one of the hardest to adapt to.
```

TicClick commented on wiki/Play_style/Double_tapping/en.md:
```diff
@@ -11,7 +11,7 @@ tags:
 *Not to be confused with [Alternating](/wiki/Play_style/Alternating).*
 
-**Double tapping** is a tapping style for [osu!standard](/wiki/Game_mode/osu!) in which players click circles, slide sliders, and spin spinners by pressing down on both of their set keys on their keyboard (`Z` and `X` by default). This style is most often used as a temporary substitute tapping style for extremely fast [stacks](/wiki/Mapping_Techniques/Stack) and [streams](/wiki/Beatmap/Pattern/St ...
+**Double tapping** is a tapping style for [osu!](/wiki/Game_mode/osu!) game mode in which players click circles, slide sliders, and spin spinners by pressing down on both of their set keys on their keyboard (`Z` and `X` by default). This style is most often used as a temporary substitute tapping style for extremely fast [stacks](/wiki/Mapping_Techniques/Stack) and [streams](/wiki/Beatmap/Pattern/ ...
```
```suggestion
**Double tapping** is a tapping style for [osu!](/wiki/Game_mode/osu!) in which players click circles, slide sliders, and spin spinners by pressing down on both of their set keys on their keyboard (`Z` and `X` by default). This style is most often used as a temporary substitute tapping style for extremely fast [stacks](/wiki/Mapping_Techniques/Stack) and [streams](/wiki/Beatmap/Pattern/Stream) that are otherwise too difficult for some players. This style is known to be the least accurate.
```

TicClick commented on wiki/Play_style/Hybrid/en.md:
```diff
@@ -9,7 +9,7 @@ tags:
 # Hybrid
 
-**Hybrid** is a tapping style for [osu!standard](/wiki/Game_mode/osu!) in which players click circles, slide sliders, and spin spinners by using a combination of [single-tapping](/wiki/Play_style/Single-tapping) and/or [alternating](/wiki/Play_style/Alternating). This style can be found from players of all skill-levels and the skill required to execute it can vary from beatmap to beatmap and vari ...
+**Hybrid** is a tapping style for [osu!](/wiki/Game_mode/osu!) game mode in which players click circles, slide sliders, and spin spinners by using a combination of [single-tapping](/wiki/Play_style/Single-tapping) and/or [alternating](/wiki/Play_style/Alternating). This style can be found from players of all skill-levels and the skill required to execute it can vary from beatmap to beatmap and va ...
```
```suggestion
**Hybrid** is a tapping style for [osu!](/wiki/Game_mode/osu!) in which players click circles, slide sliders, and spin spinners by using a combination of [single-tapping](/wiki/Play_style/Single-tapping) and/or [alternating](/wiki/Play_style/Alternating). This style can be found from players of all skill-levels and the skill required to execute it can vary from beatmap to beatmap and variation to variation.
```

TicClick commented on wiki/Play_style/Single-tapping/en.md:
```diff
@@ -10,7 +10,7 @@ tags:
 # Single-tapping
 
-**Single-tapping** is a tapping style for [osu!standard](/wiki/Game_mode/osu!) in which players click circles, slide sliders, and spin spinners using only one key on their keyboard (`Z` or `X` by default). This style is common amongst players of all skill levels and is the easiest to adapt to.
+**Single-tapping** is a tapping style for [osu!](/wiki/Game_mode/osu!) game mode in which players click circles, slide sliders, and spin spinners using only one key on their keyboard (`Z` or `X` by default). This style is common amongst players of all skill levels and is the easiest to adapt to.
```
```suggestion
**Single-tapping** is a tapping style for [osu!](/wiki/Game_mode/osu!) in which players click circles, slide sliders, and spin spinners using only one key on their keyboard (`Z` or `X` by default). This style is common amongst players of all skill levels and is the easiest to adapt to.
```

TicClick commented on wiki/Play_style/Tapping_style/en.md:
```diff
@@ -10,4 +10,4 @@ tags:
 *Not to be confused with [Play style](/wiki/Play_style).*
 
-A **tapping style** is a term used to describe a method of tapping keys/sending inputs on one's input device to tap [hit objects](/wiki/Hit_objects) in osu!. Common tapping styles for [osu!standard](/wiki/Game_mode/osu!) include, [alternating](/wiki/Play_style/Alternating), [single-tap](/wiki/Play_style/Single-tap), [double tap](/wiki/Play_style/Double_tapping), [Hybrid](/wiki/Play_style/Hybrid), ...
+A **tapping style** is a term used to describe a method of tapping keys/sending inputs on one's input device to tap [hit objects](/wiki/Hit_objects) in osu!. Common tapping styles for [osu!](/wiki/Game_mode/osu!) game mode, include [alternating](/wiki/Play_style/Alternating), [single-tap](/wiki/Play_style/Single-tap), [double tap](/wiki/Play_style/Double_tapping), [Hybrid](/wiki/Play_style/Hybrid ...
```
```suggestion
A **tapping style** is a term used to describe a method of tapping keys/sending inputs on one's input device to tap [hit objects](/wiki/Hit_objects) in osu!. Common tapping styles for [osu!](/wiki/Game_mode/osu!) include [alternating](/wiki/Play_style/Alternating), [single-tap](/wiki/Play_style/Single-tap), [double tap](/wiki/Play_style/Double_tapping), [Hybrid](/wiki/Play_style/Hybrid), and [tapx](/wiki/Play_style/Tapx).
```

TicClick commented on wiki/Play_style/Tapx/en.md:
```diff
@@ -12,7 +12,7 @@ tags:
 *Notice: This play style requires the use of a graphics tablet.*
 
-**Tapx** is a tapping style for [tablet-only](/wiki/Play_style#tablet-only) players in [osu!standard](/wiki/Game_mode/osu!) in which players click circles, slide sliders, and spin spinners by tapping a tablet pen onto a tablet, along with a secondary keyboard and/or tablet key to assist with [streams](/wiki/Beatmap/Pattern/Stream) and [stacks](/wiki/Mapping_Techniques/Stack). This method is fairl ...
+**Tapx** is a tapping style for [tablet-only](/wiki/Play_style#tablet-only) players in [osu!](/wiki/Game_mode/osu!) game mode in which players click circles, slide sliders, and spin spinners by tapping a tablet pen onto a tablet, along with a secondary keyboard and/or tablet key to assist with [streams](/wiki/Beatmap/Pattern/Stream) and [stacks](/wiki/Mapping_Techniques/Stack). This method is fai ...
```
```suggestion
**Tapx** is a tapping style for [tablet-only](/wiki/Play_style#tablet-only) players in [osu!](/wiki/Game_mode/osu!) in which players click circles, slide sliders, and spin spinners by tapping a tablet pen onto a tablet, along with a secondary keyboard and/or tablet key to assist with [streams](/wiki/Beatmap/Pattern/Stream) and [stacks](/wiki/Mapping_Techniques/Stack). This method is fairly difficult for most players and is not very popular amongst most tablet players.
```

TicClick commented on wiki/Project_Loved/en.md:
```diff
@@ -27,7 +27,7 @@ There are some very minimal criteria that need to be met for beatmaps to be nomi
 - Its creator is neither banned nor restricted
 - At least one difficulty can be passed
-- It has at least 30 favourites (only applies to osu!standard maps)
+- It has at least 30 favourites (only applies to osu! game mode maps)
```
```suggestion
- It has at least 30 favourites (only applies to osu! maps)
```

TicClick commented on wiki/Projects/fr.md:
```diff
@@ -224,7 +224,7 @@ Outils et/ou services créés par la communauté osu!. Vous êtes libres d'ajout
 - [BloodyFox \[Android\]](https://osu.ppy.sh/community/forums/topics/475693)—Rechercher et télécharger des beatmaps/Voir les statistiques d'un joueur.
 - [Detect the osu! folder easily](https://osu.ppy.sh/community/forums/topics/381311)—Trouve le dossier osu! et rapporte son chemin. Utile pour les développeurs travaillant en IO sur leurs applications
-- [o!subscribe](https://osu.ppy.sh/community/forums/topics/587110)—Un service qui vous permet de vous abonner à vos créateurs favoris grâce à Discord.
+- [osu!ubscribe](https://osu.ppy.sh/community/forums/topics/587110)—Un service qui vous permet de vous abonner à vos créateurs favoris grâce à Discord.
```
this is the utility's name, please read into what you are changing
```suggestion
- [o!subscribe](https://osu.ppy.sh/community/forums/topics/587110)—Un service qui vous permet de vous abonner à vos créateurs favoris grâce à Discord.
```

TicClick commented on wiki/Projects/en.md:
```diff
@@ -224,7 +224,7 @@ Tools and/or services made by the osu! community. Feel free to add your own osu!
 - [BloodyFox \[Android\]](https://osu.ppy.sh/community/forums/topics/475693)—Search and view user stats and search and download beatmaps.
 - [Detect the osu! folder easily](https://osu.ppy.sh/community/forums/topics/381311)—Finds the osu! folder and report its path. Intended for devs working on IO related tasks in their applications.
-- [o!subscribe](https://osu.ppy.sh/community/forums/topics/587110)—A service that allows you to subscribe to your favorite creators using Discord.
+- [osu!ubscribe](https://osu.ppy.sh/community/forums/topics/587110)—A service that allows you to subscribe to your favorite creators using Discord.
```
```suggestion
- [o!subscribe](https://osu.ppy.sh/community/forums/topics/587110)—A service that allows you to subscribe to your favorite creators using Discord.
```

TicClick commented on wiki/Ranking_Criteria/osu!/en.md:
```diff
@@ -2,7 +2,7 @@
 ***Note: This document is an extension of the [general ranking criteria](/wiki/Ranking_Criteria).***
 
-This set of **osu! ranking criteria** lays out [rules and guidelines](/wiki/Ranking_Criteria#general-terms) that [osu!standard](/wiki/Game_mode/osu!)-specific [beatmaps](/wiki/Beatmap) must follow in order to progress through the [beatmap ranking procedure](/wiki/Beatmap_ranking_procedure).
+This set of **osu! ranking criteria** lays out [rules and guidelines](/wiki/Ranking_Criteria#general-terms) that [osu!](/wiki/Game_mode/osu!)game mode-specific [beatmaps](/wiki/Beatmap) must follow in order to progress through the [beatmap ranking procedure](/wiki/Beatmap_ranking_procedure).
```
```suggestion
This set of **osu! ranking criteria** lays out [rules and guidelines](/wiki/Ranking_Criteria#general-terms) that [osu!](/wiki/Game_mode/osu!)-specific [beatmaps](/wiki/Beatmap) must follow in order to progress through the [beatmap ranking procedure](/wiki/Beatmap_ranking_procedure).
```

TicClick commented on wiki/Score/en.md:
```diff
@@ -172,11 +172,11 @@ Combo will not be lost for missing droplets (considered as *Miss Droplet* in ser
 - 300s and a combo increment are not given to a finished spinner
 - reverse slider ticks are counted as a full score of a hit
-  - note that in osu!standard, a reverse tick only gives 30 points
+  - note that in the osu!, a reverse tick only gives 30 points
```
```suggestion
  - note that in osu!, a reverse tick only gives 30 points
```

TicClick commented on wiki/Skinning/History/en.md:
```diff
@@ -349,7 +349,7 @@ Notes:
 Notes:
 
-- This is for [osu!standard](/wiki/osu!standard) only.
+- This is for [osu!](/wiki/Game_modes/osu!) game mode only.
```
```suggestion
- This is for [osu!](/wiki/Game_modes/osu!) only.
```

TicClick commented on wiki/Skinning/skin.ini/en.md:
```diff
@@ -54,7 +54,7 @@ If your `skin.ini` does not specify a `Version`, it will default to this version
 
 - Thumbnail support.
-  - Must be enabled by the user in the [Options](/wiki/Options) and be supported by the skin in the [skin.ini](/wiki/skin.ini) file.
+  - Must be enabled by the user in the [Options](/wiki/Options) and be supported by the skin in the skin.ini file.
```
```suggestion
  - Must be enabled by the user in the [Options](/wiki/Options) and be supported by the skin in the [skin.ini](/wiki/Skinning/skin.ini) file.
```

TicClick commented on wiki/Welcome/da.md:
```diff
@@ -13,13 +13,13 @@ Velkommen til osu!, et free-to-win rytmespil udviklet af peppy komplet med fire
 - [Multiplayer](/wiki/Multi)
 - [Rangeringskriterier](/wiki/Ranking_Criteria)
-  - ![osu!standard icon](/wiki/shared/mode/osu.png) [osu!standard](/wiki/Ranking_Criteria/osu!standard)
+  - ![osu! icon](/wiki/shared/mode/osu.png) [osu!s](/wiki/Ranking_Criteria/osu!)
```
```suggestion
  - ![osu! icon](/wiki/shared/mode/osu.png) [osu!](/wiki/Ranking_Criteria/osu!)
```

TicClick reviewed (changes requested):
some things to consider:
- split your changes to make the review process easier
- proof-read your changes
- make sure you understand what is the goal. I've had to turn down some significant amount of your changes, which is sad

I'll apply the changes myself, squash the commits and merge the request afterwards

peppy commented:
Please avoid force pushing branches, makes it very hard to know exactly what has happened.

mousesalt commented:
i have been noticed https://github.com/ppy/osu-wiki/blob/master/wiki/Guides/osu!mania_Mapping_Guide/id.md got wiped almost whole page, what did happened here

DoryanR commented:
On the quoted article, only the term `o!m` has been changed, the rest of the article remains unchanged.

However, from what I see, the article seems outdated compared to the English version.

mousesalt commented:
> However, from what I see, the article seems outdated compared to the English version.
I don't think it's outdated since the translation is pretty match. Do you know how to revert this back? because this was translated newest version already

DoryanR commented:
It seems to me to be quite outdated, because if we compare the Indonesian article and the English one, there is a lot of information missing (if we are talking about the [`osu!mania Mapping Guide`](https://github.com/ppy/osu-wiki/blob/master/wiki/Guides/osu!mania_Mapping_Guide/en.md) article of course)

I can always go back on this modification, but as it only modifies a small part of the article (just the conversion of the terms `o!m` > `osu!mania`), it won't make much sense.

![](https://i.imgur.com/qSwjolr.png)

I also looked at the changes that were made to the Indonesian version, and none of the changes involve deleting any part of the article, either on the other commits or the one that was made.

That's what makes me say that the article has not been modified in this way. Afterwards, as there was a forcepush, we may not have all the changes, and I have a good memory that personally, I just made this kind of change of terms on all the articles outside the English file.

TicClick commented:
I don't think it's been wiped:

```
blah% git log --oneline -3 wiki/Guides/osu\!mania_Mapping_Guide/id.md
fa7fd689f Convert `osu!standard` > `osu!` everywhere
f38c410d0 Fix formatting in osu!mania mapping guide
9dc4a2c1a Use atx headers instead of Setext
```

output of `git show fa7fd689f wiki/Guides/osu\!mania_Mapping_Guide/id.md`:

```diff
commit fa7fd689fe7daa1c750c7cb215d10a65b4600924
Author: Doryan <55928127+DoryanR@users.noreply.github.com>
Date:   Wed Apr 14 09:18:27 2021 +0200

    Convert `osu!standard` > `osu!` everywhere

diff --git a/wiki/Guides/osu!mania_Mapping_Guide/id.md b/wiki/Guides/osu!mania_Mapping_Guide/id.md
index 07f0a9f72..04cecfb27 100644
--- a/wiki/Guides/osu!mania_Mapping_Guide/id.md
+++ b/wiki/Guides/osu!mania_Mapping_Guide/id.md
@@ -4,7 +4,7 @@

 ### Apa itu Mania?

-Jadi apa sebenarnya Mania? Banyak dari Anda Osu! veteran di luar sana yang mungkin sudah tahu apa itu Mania. Osu!mania adalah salah satu dari 4 mode permainan osu!. Banyak orang yang pertama kali melihat o!m berpikir "Oh keren! Guitar Hero!". Dan memang, osu!mania sangat mirip dengan Guitar Hero. Kedua game ini disebut-"VSRGs", atau **Vertical Scrolling Rhythm Game**. Yang berarti seperti namanya, bahwa note musik yang melaju ke atas maupun ke bawah secara vertikal. Ada banyak permainan yang mirip dengan o!m seperti "Stepmania", "O2Jam" atau "Beatmania IIDX". Tapi untuk saat ini, kami akan berkonsentrasi hanya pada osu!mania!
+Jadi apa sebenarnya Mania? Banyak dari Anda Osu! veteran di luar sana yang mungkin sudah tahu apa itu Mania. Osu!mania adalah salah satu dari 4 mode permainan osu!. Banyak orang yang pertama kali melihat osu!mania berpikir "Oh keren! Guitar Hero!". Dan memang, osu!mania sangat mirip dengan Guitar Hero. Kedua game ini disebut-"VSRGs", atau **Vertical Scrolling Rhythm Game**. Yang berarti seperti namanya, bahwa note musik yang melaju ke atas maupun ke bawah secara vertikal. Ada banyak permainan yang mirip dengan osu!mania seperti "Stepmania", "O2Jam" atau "Beatmania IIDX". Tapi untuk saat ini, kami akan berkonsentrasi hanya pada osu!mania!
```
